### ISSUE TITLE
Implement Campaign Info Brick

### ISSUE THREAD

**Issue by Pashaminkovsky** [pixiebrix/pixiebrix-extension#9380]
The goal is to create a Campaign Info brick similar to the Fetch Me brick. This brick would expose the same data as the campaign members endpoint. 

One caveat is around permissions -- campaigns currently require an Admin or Manager role to access. This customer wants mods deployed to end users to have access to the campaign data. One workaround we can implement is to have a service account with Manager permissions be the auth for this request. End users in this client don't have access to dev tools or the page editor, so it's not a major security concern.

Spike: https://github.com/pixiebrix/pixiebrix-extension/issues/9337

**Comment by Pashaminkovsky** [comment]
Yes we'd be returning campaigns that the user is a member of. Good callout I updated the description. cc @mthek10

**Comment by fungairino** [comment]
Came from a quick chat with Pasha over Zoom - originally I had thought this was just requesting a brick that could fetch a given organization campaign data (using an admin service account token). 

However it seems like this might be a bigger ask, potentially fetching multiple campaigns scoped to just the user and organization. Pasha is going to look into nailing down the exact specifics on what data we need to fetch and move this back to a spike (we potentially will have to modify or create new endpoints for this specific data)

**Comment by twschiller** [comment]
If it's exposed as a new endpoint on app, the brick could just be defined in YAML (as a wrapper around the HTTP call to the endpoint). We'd probably just expose as `/api/me/campaigns/` or similar
